terracumber_config/tf_files/SUSEManager-4.3-NUE.tf

```diff
@@ -102,7 +102,7 @@ module "cucumber_testsuite" {
   cc_username = var.SCC_USER
   cc_password = var.SCC_PASSWORD
 
-  images = ["centos7o", "opensuse155o", "sles15sp4o", "ubuntu2204o"]
+  images = ["rocky8o", "opensuse155o", "sles15sp4o", "ubuntu2204o"]
```

Comment:
Do we also need to perform the changes in our mirrors?

Reply:
Rocky 8 images are currently kept in sync on our mirror.

Apparently, we were just missing to sync the RES8-SUSE-Manager-Tools updates for 4.3 CI.
That has been handled in a previous PR to the infra repo ;)

Reply:
ok